unconvert.go

```diff
@@ -200,21 +201,14 @@ func main() {
 		}
 		wg.Wait()
 	} else {
-		var files []string
-		for f := range m {
-			files = append(files, f)
-		}
-		sort.Strings(files)
-		found := false
-		for _, f := range files {
-			if len(m[f]) != 0 {
-				found = true
+		var edits positionSlice
+		for _, positions := range m {
+			for pos := range positions {
+				edits = append(edits, pos)
 			}
-			print(f, m[f])
-		}
-		if found {
-			os.Exit(1)
 		}
+		sort.Sort(edits)
```

Comment:
things no longer sort naturally by line number as they did before #12, because we're not getting it for free from `AppendTo` (https://github.com/mdempsky/unconvert/pull/12/files#diff-d45c7d690c87bd02c6ddd98b0dbd510fL119) as before. So now all the edits within a file are unordered without change.